### DISCUSSION TITLE
Automatic on/off switch for rules

### DISCUSSION THREAD

**Discussion by Aint-Nobody** [arendst/Tasmota#15745] [Q&A] [answered]
It would be very nice if there where an option, that a rule will be only executed if a given IP address is reachable.
So if my smartphone IP address in my home wifi is reachable it indicates if I am at home or not.
So turns a light at a special time on or not.
Could it be realize ?

**Comment by barbudor** [comment, marked as answer]
https://tasmota.github.io/docs/Commands/#ping

You can have a rule with Ping to find out if your phone is here or not, set a varaible accordingly and use that variable if a rule with IF

**Reply by Aint-Nobody** [reply]
Thx4 answer
But in the ready build bin is the command use_ping not compiled 🙁

**Reply by barbudor** [reply]
Yes
https://tasmota.github.io/docs/Compile-your-build/
